pkgs/development/compilers/llvm/4/clang/default.nix

```diff
@@ -29,12 +29,23 @@ let
       sed -i -e 's/DriverArgs.hasArg(options::OPT_nostdlibinc)/true/' lib/Driver/ToolChains.cpp
     '';
 
+    outputs = [ "out" "python" ];
```

Comment:
Are we splitting outputs by default nowadays? Should I include other outputs here?